staging/knative/charts/eventing/templates/:vsp controller.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
```

Comment:
the name of this filename contains a typo

Reply:
A fellow vim user :+1:

Reply:
Hmm I thought I fixed this... guessed the fix was on another draft branch :(

Reply:
Fixed. Forgot to push :(